src/bun.js/javascript.zig

```diff
@@ -1793,6 +1793,13 @@ pub const VirtualMachine = struct {
         );
         this.eventLoop().ensureWaker();
 
+        // Clear the terminal
+        if (!strings.eqlComptime(this.bundler.env.map.get("BUN_CONFIG_NO_CLEAR_TERMINAL_ON_RELOAD") orelse "0", "true")) {
```

Comment:
I kinda copied this from the `bun.reloadProcess()` function in https://github.com/oven-sh/bun/blob/main/src/bun.zig#L1277.

Reply:
This will cause it to clear the terminal every time bun starts, whenever a Worker starts, and whenever a macro is called


Instead, put it here: https://github.com/oven-sh/bun/blob/8fccc1d27dd4b43833367b47973c65d1a44ce563/src/bun.js/javascript.zig#L629

Reply:
Thank you 🙂